content/html/concepts/attributes/attributes.md

```diff
@@ -30,29 +30,27 @@ So for example, the `<img>` image element has a mandatory `src` attribute that c
 - `src` is the attribute name.
 - `codecademy_logo.png` is the attribute value.
 
-All HTML elements can have attributes. Multiple of them, too!
+All HTML elements can have attributes; many have more than one.
```

Comment:
```suggestion
All HTML elements can have attributes and many have more than one.
```

Reply:
I think two clauses, as I had it, is better here. At the very least I would suggest a compromise with a comma before the "and":

> All HTML elements can have attributes, and many have more than one.

Reply:
Looking over this, I'm good with your initial change if we want to separate the points. Otherwise, I would suggest "All HTML elements can have one or more attributes" or just "HTML elements can have one or more attributes."  Lmk what you 🧠  think!

I'm also seeing now that the following code snippet is not prefaced so I'm going to suggest that we add on "The example below displays an image element with three attributes:"

Reply:
Done!